go/vt/vtgate/evalengine/testcases/cases.go

```diff
@@ -1816,6 +1817,26 @@ func FnFromDays(yield Query) {
 	}
 }
 
+func FnTimeToSec(yield Query) {
+	for _, d := range inputConversions {
+		yield(fmt.Sprintf("FROM_DAYS(%s)", d), nil)
+	}
+
+	time := []string{
+		`0`,
+		`'00:00:00'`,
+		`'22:23:00'`,
+		`'00:39:38'`,
+		`'000220`,
+		`'2003-09-03 00:39:38'`,
+		`'2003-09-03'`,
```

Comment:
We should also have a `TIME` instance here that's larger than 24 hours to make sure that works properly as well.

Reply:
done.